pkg/server/server.go

```diff
@@ -0,0 +1,51 @@
+package server
+
+import (
+	"context"
+	"log/slog"
+	"net/http"
+	"strconv"
+
+	camelv1alpha "github.com/apache/camel-k/pkg/apis/camel/v1alpha1"
+	"github.com/gin-gonic/gin"
+	"github.com/sco1237896/sco-backend/pkg/client"
+)
+
+var (
+	logger = slog.Default()
```

Comment:
you can create a logger with some additional context, i.e. `slog.Default().With(slog.String("component", "server")` if that make sense.

Reply:
https://github.com/sco1237896/sco-backend/pull/1/commits/dde65cf334e96abe6a4b9127b354cc64d47da7bc